www/js/history/review-controller.js

```diff
@@ -2,13 +2,50 @@ angular.module('starter.review', [])
 
 .controller('ReviewCtrl', function($scope, $stateParams) {
 
+	$scope.newReview = {};
+
+// START: Five star rating thang
+	// Initialization code
+	var fiveStarElement = document.getElementById("fiveStarRating");
+	var fourStarElement = document.getElementById("fourStarRating");
+	var threeStarElement = document.getElementById("threeStarRating");
+	var twoStarElement = document.getElementById("twoStarRating");
+	var oneStarElement = document.getElementById("oneStarRating");
```

Comment:
I use a list/array to activate/deactivate the appropriate star elements based on the selection. Grabbing each element once and referencing them in the list seemed better than doing it on each pass. Thoughts?